spec/takeaway_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'takeaway'
+
+describe Takeaway do
+  let(:order) { double :order, as_hash: {'Spring Rolls' => 2, 
+    'Crispy Duck' => 1}, total: 12.97 }
+  let(:order_klass) { double :order_klass, new: order }
+  let(:carrot_order) { double :order, as_hash: {'Carrot' => 99} }
+  let(:carrot_order_klass) { double :order_klass, new: carrot_order }
+
+  let(:phone) { double :phone, complete_order: nil }
+  let(:menu) { double :menu, raw: {'Spring Rolls' => 2.99,
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.